noisyatom/catalog/models.py

```diff
@@ -104,9 +112,11 @@ class Meta:
 
     def __unicode__(self):
         return self.name
-
+'''
```

Comment:
Same as before. Why commented out?

Reply:
Thanks! I did remove the old code. It won't work anymore with Django 2.2.